lib/indico.js

```diff
@@ -60,11 +64,12 @@ var make_request = function (name, data, batch, config) {
 
   body = addKeywordArguments(body, config);
   apis = config ? config['apis'] : false
+  version = config ? config['version'] : false
```

Comment:
I think we did it this way in node because we were doing everything in config. so it sort of makes sense for version to go in as well. otherwise, everywhere else we just put it in the function signature like api and cloud and batch.